### DISCUSSION TITLE
Document migration between deployment targets

### DISCUSSION THREAD

**Discussion by suchy** [remix-run/remix#4908] [Proposals]
Hi,

currently, when creating a new app I can choose a deploy target. App says "Choose Remix App Server if you're unsure; it's easy to change deployment targets." but this process is not documented.

Improving that part of the documentation might be helpful during Remix research/experiments/PoC.

What do you think?

**Comment by suchy** [comment]
@ryanflorence I watched route map planning, where you mentioned this proposal.

I thought about that and it would be nice to see in the documentation, which parts could change when migrating to other providers. Something like "hey, when migrating to a different provider, you might take a look on entry.server, server, and config (that's it?)"

I would not expect full vercel/netlify/deno/fly documentation ;-)

**Reply by runofthemill** [reply]
+1 - watching roadmap planning #3 right now and definitely see the concern over having to write docs for potentially numerous javascript servers, but also think a "here's what to look for" section/graf could be a very useful & valuable addition, but with a much smaller scope.

another improvement could be indicating in starters/examples what files are specific to the intended deploy target. potentially could add a bit of overhead in some scenarios, but like in the Indie Stack saying something along the lines of 

> Starting with this stack but not using Fly.io or Docker with it? You won't need `start.sh`, `Dockerfile`, `.dockerignore`, `fly.toml`, etc.
